index.js

```diff
@@ -201,31 +247,152 @@ class LevitonDecoraSmartPlatform {
     // get device and token out of context to update status
     const device = accessory.context.device
     const token = accessory.context.token
-    const status = await this.getStatus(device, token)
-
-    // get the accessory service, if not add it
-    const service =
-      accessory.getService(Service.Lightbulb, device.name) ||
-      accessory.addService(Service.Lightbulb, device.name)
-
-    // add handlers for on/off characteristic, set initial value
-    service
-      .getCharacteristic(Characteristic.On)
-      .on('get', this.onGetPower(service, device, token).bind(this))
-      .on('set', this.onSetPower(service, device, token).bind(this))
-      .updateValue(status.power === 'ON' ? true : false)
-
-    // set handlers for brightness, set initial value and min/max bounds
-    service
-      .getCharacteristic(Characteristic.Brightness)
-      .on('get', this.onGetBrightness(service, device, token).bind(this))
-      .on('set', this.onSetBrightness(service, device, token).bind(this))
-      .setProps({
-        minValue: status.minLevel,
-        maxValue: status.maxLevel,
-        minStep: 1,
-      })
-      .updateValue(status.brightness)
+    
+    //Get the model number
+    this.log('  -Device Model: ', device.model)
+    
+    
+    switch (device.model) {
```

Comment:
This switch should take advantage of case fall through to make it more compact and easy to read. 

> This is an example of a single operation sequential case statement, where four different values perform exactly the same.
```
var Animal = 'Giraffe';
switch (Animal) {
  case 'Cow':
  case 'Giraffe':
  case 'Dog':
  case 'Pig':
    console.log('This animal will go on Noah\'s Ark.');
    break;
  case 'Dinosaur':
  default:
    console.log('This animal will not.');
}
```
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/switch

Reply:
Just issued another commit with your suggestion on the case fall through.  Thanks for that pointer.